bus.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+function Product(name, filePath, imageId) {
+  this.name = name ;
+  this.filePath= filePath;
+  this.imageId = imageId;
+  this.displayed = 0;
+  this.clicks = 0;
+}
+
+var photosOnSecondToLastScreen = [];
+var photosOnPreviousScreen = [];
+var photosOnScreen = [];
+
+var bag =  new Product('Bag','img/bag.jpg');
+var banana =  new Product('Banana Cutter', 'img/banana.jpg');
+var bathroom =  new Product ('Tablet Holder for Bathroom', 'img/bathroom.jpg');
+var boots =  new Product('Boots', 'img/boots.jpg');
+var breakfast =  new Product('breakfast', 'img/breakfast.jpg');
+var bubblegum =  new Product('Bubble Gum', 'img/bubblegum.jpg');
+var chair =  new Product ('Comfy Chair', 'img/chair.jpg');
+var cthulhu =  new Product('Cthulhu centerpeice', 'img/cthulhu.jpg');
+var duckDog =  new Product('duck dog', 'img/dog-duck.jpg');
+var dragon =  new Product ('Dragon', 'img/dragon.jpg');
+var pen =  new Product('pen', 'img/pen.jpg');
+var broomDog =  new Product('Broom Dog', 'img/pet-sweep.jpg');
+var scissors =  new Product('scissors', 'img/scissors.jpg');
+var shark =  new Product('Shark', 'img/shark.jpg');
+var broomBaby =  new Product ('Broom Baby', 'img/sweep.png');
+var tauntaun =  new Product ('tauntaun', 'img/tauntaun.jpg');
+var unicorn =  new Product ('unicorn', 'img/unicorn.jpg');
+var usb =  new Product ('USB', 'img/usb.gif');
+var watering =  new Product ('Watering Can', 'img/water-can.jpg');
+var wine =  new Product ('Wine Glass', 'img/wine-glass.jpg');
+
+var photos = [bag, banana, bathroom, boots, breakfast, bubblegum, chair, cthulhu, duckDog, dragon, pen, broomDog, scissors, shark, broomBaby, tauntaun, unicorn, usb, watering, wine ];
+
+try {
+  photos = JSON.parse(localStorage.photos);
+} catch(error){
+  console.log('error retreiveing local storage');
+}
+
+var clicks = 0;
+
+var getRandomIndex = function() {
```

Comment:
Why create a function expression (var) instead of function declaration (like the rest of your code) here?  This is inconsistent with the rest of your code.